lib/win32/backend/policies.rb

```diff
@@ -0,0 +1,44 @@
+#
+# Author:: Piyush Awasthi (<piyush.awasthi@msystechnologies.com>)
+# Copyright:: Copyright (c) 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Win32::Backend::Policies
+
+  # Validate import certificate file format
+  def validate_args(import_cert_params)

```

Comment:
Right now this method is not being used, but I suppose this will be used while `certificate add` functionality. Since this validates the certificate, would it be good to name it as `validate_certificate` as we have `validate_store` below?